source/examples/rapids-coiled-cudf/notebook.ipynb

```diff
@@ -0,0 +1,2461 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    taxi_zones = pd.read_csv("taxi_zones.csv")
</pre><p>when you download the file, the name of the file has a different name, it's called <code>taxi_zone_lookup.csv</code> you will probably want to be consistent with that.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>